content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -85,7 +85,7 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    chmod +x kubectl
    mkdir -p ~/.local/bin/kubectl
    mv ./kubectl ~/.local/bin/kubectl
-   # and then append (or prepend) ~/.local/bin to $PATH
+   # and then append (or prepend) ~/.local/bin/kubectl to $PATH
```

Comment:
I think that "what to append (or prepend) to $PATH" is
`~/.local/bin` (the directory where the `kubectl` binary exists),
not `~/.local/bin/kubectl` (the path to `kubectl` binary itself).

So I suggest to keep as-is (`# and then append (or prepend) ~/.local/bin to $PATH`).

Reply:
`~/.local/bin/kubectl` is not a folder, it is the new path for the `kubectl` binary.
`~/.local/bin` is the folder to be added to `$PATH`.

Reply:
Agree that `~/.local/bin` is the folder to be added to `$PATH`
`~/.local/bin/kubectl` is the binary not the path